.eslintrc.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
```

Comment:
Rules for the eslint, for now I based it on the airbnb one. similar to this : https://itnext.io/how-to-setup-typescript-eslint-prettier-and-react-in-5-minutes-44cfe8af5081

We can setup our own rules ( this would avoids the boiler plate on our side )